### ISSUE TITLE
`Learn more about your data` opening `#brave-today` rather than `#brave-news`

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#22553]
## Description 

Tapping on `Learn more about your data` via the Brave News opt-in card opens `https://brave.com/privacy/browser/#brave-today`. However, it should be `https://brave.com/privacy/browser/#brave-news` as we've re-branded from `Today` -> `News` a while ago.

## Steps to reproduce 

   1. download a version of Brave that has the Brave News opt-in card visible (`1.38.x` or higher)
   2. tap on `Learn more about your data` and note the link/anchor being referred

## Actual result 

Opens `https://brave.com/privacy/browser/#brave-today`

**`Example`** | **`Example`**
----------------|-----------------
![Screenshot_20220426-010958_Brave - Nightly](https://user-images.githubusercontent.com/2602313/165226474-eb06f5c6-af80-45eb-9b01-077d2d47fddc.jpg) | ![Screenshot_20220426-003907_Brave - Nightly](https://user-images.githubusercontent.com/2602313/165226489-ca2fd428-549c-4246-a11e-6ec1418eab53.jpg)

## Expected result

Should be opening `https://brave.com/privacy/browser/#brave-news`

## Issue reproduces how often 

100% reproducible using the STR/Cases outlined above.

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? `No` (present in `1.38.x` or higher)
- Can you reproduce this issue with the current Play Store Beta version? `Yes`
- Can you reproduce this issue with the current Play Store Nightly version? `Yes`

## Device details
   - Install type (ARM, x86): `ARM`
   - Device type (Phone, Tablet, Phablet): `Samsung S10+`
   - Android version: `Android 12`
   
## Brave version

* `1.40.12 Chromium: 101.0.4951.41`

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? `N/A`
- Does the issue resolve itself when disabling Brave Rewards? `N/A`
- Is the issue reproducible on the latest version of Chrome? `N/A`

**Comment by kjozwiak** [comment]
We have a similar issue on desktop where we're opening `https://brave.com/privacy/browser` but not using the `#brave-news` anchor. We should be opening  `https://brave.com/privacy/browser/#brave-news` as per the above.

**Comment by stephendonner** [comment]
@kjozwiak can we remove `OS/Desktop` since the commit here only addressed the Android Java code?  I've filed https://github.com/brave/brave-browser/issues/23587 to take care of desktop 👍

**Comment by kjozwiak** [comment]
> @kjozwiak can we remove `OS/Desktop` since the commit here only addressed the Android Java code? I've filed #23587 to take care of desktop 👍

yup, removed 👍

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Pixel 6` running `Android 13` using the following build(s):

```
Brave | 1.41.82 Chromium: 103.0.5060.66 (Official Build) beta (32-bit)
--- | ---
Revision | 20b1569438a85e631d15e83eb355e3e326e5da6f-refs/branch-heads/5060@{#1066}
OS | Android 13;
```

Went through the STR/Cases outlined via https://github.com/brave/brave-browser/issues/22553#issue-1215405696 and ensured that `https://brave.com/privacy/browser/#brave-news` was being opened when tapping on `Learn more about your data`.
